src/Bridge/Doctrine/Orm/Extension/FilterEagerLoadingExtension.php

```diff
@@ -37,8 +38,12 @@ public function __construct(ResourceMetadataFactoryInterface $resourceMetadataFa
     /**
      * {@inheritdoc}
      */
-    public function applyToCollection(QueryBuilder $queryBuilder, QueryNameGeneratorInterface $queryNameGenerator, string $resourceClass, string $operationName = null)
+    public function applyToCollection(QueryBuilder $queryBuilder, QueryNameGeneratorInterface $queryNameGenerator, string $resourceClass = null, string $operationName = null, array $context = [])
     {
+        if (null === $resourceClass) {
+            throw new InvalidArgumentException('The "$resourceClass" parameter must not be null');
+        }
```

Comment:
Weird no? How is this possible and why do we change the interface to allow null? Especially if `supports` method does require this argument can't it be mandatory?

Reply:
It's only possible to defined **optional** parameters when overriding the signature of method defined in a parent interface. It's why.

Reply:
Sure. Once the parent interface argument becomes mandatory we will remove this test right?

Reply:
Indeed, but I'm not sure it is worth it to make the argument mandatory in a future version (pain to upgrade for almost no gain).